MANIFEST.in

```diff
@@ -6,3 +6,30 @@ recursive-include sklearn/datasets *.csv *.csv.gz *.rst *.jpg *.txt *.arff.gz *.
 include COPYING
 include README.rst
 include pyproject.toml
+include .binder/postBuild
+include .binder/requirements.txt
+include .circleci/artifact_path
+include .circleci/config.yml
+include .mailmap
+include sklearn/externals/README
+include sklearn/svm/src/liblinear/COPYRIGHT
+include sklearn/svm/src/libsvm/LIBSVM_CHANGES
+
+# added by check_manifest.py
+include *.in
+include *.md
+include *.py
+include *.yaml
+include *.yml
+include .coveragerc
+include Makefile
+recursive-include asv_benchmarks *.json
+recursive-include asv_benchmarks *.py
+recursive-include benchmarks *.py
+recursive-include build_tools *.cmd
+recursive-include build_tools *.py
+recursive-include build_tools *.sh
+recursive-include build_tools *.yml
+recursive-include build_tools Makefile
+recursive-include maint_tools *.py
+recursive-include maint_tools *.sh
```

Comment:
I do not think we want to include benckmark scripts + build tools in the wheel.

Reply:
ok I'll add them to ignore list, there are few more suggestion from the check-manifest [here](https://github.com/nixphix/scikit-learn/runs/1051197279?check_suite_focus=true), I ll add them as well